app/pings.yaml

```diff
@@ -43,3 +43,14 @@ topsites-impression:
     - https://github.com/mozilla-mobile/fenix/pull/23945
   notification_emails:
     - android-probes@mozilla.com
+
+spoc:
+  description: |
+    Contains data about user's interactions with Pocket sponsored stories.
```

Comment:
Can we be a bit more specific about what this is about? (e.g. impressions?)

Reply:
I updated the ping description to
>    Contains data identifying with which Pocket sponsored story the user
>    interacted with and the type of interaction: story impression or click.